python/tank/util/user_settings.py

```diff
@@ -36,101 +42,168 @@ def _init_singleton(self):
         """
         Singleton initialization.
         """
-        path = self._compute_config_location()
-        logger.debug("Reading user settings from %s" % path)
+        self._path = self._compute_config_location()
+        logger.debug("Reading user settings from %s" % self._path)
 
-        self._user_config = self._load_config(path)
+        self._user_config = self._load_config(self._path)
 
         # Log the default settings
-        logger.debug("Default site: %s" % (self.default_site or "<missing>",))
-        logger.debug("Default login: %s" % (self.default_login or "<missing>",))
+        logger.debug("Default site: %s", self._to_friendly_str(self.default_site))
+        logger.debug("Default login: %s", self._to_friendly_str(self.default_login))
```

Comment:
why the inconsistent usage of `,` and `%`? You first do 

```
logger.debug("Reading user settings from %s" % self._path)
```

but then proceed to change a lot of messages to be

```
logger.debug("Default site: %s", self._to_friendly_str(self.default_site))
```

What's wrong with `%`? That's what we consistently use throughout most of code base? Consistency is important.